velox/common/base/tests/ScopedLockTest.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/common/base/ScopedLock.h"
+
+#include <gtest/gtest.h>
+
+namespace facebook {
+namespace velox {
+
+TEST(ScopedLockTest, basic) {
```

Comment:
for completeness, maybe have a test adding multiple callbacks as well?

Reply:
Good point. Thanks!